### PR TITLE
Added prototype plan notes from meeting at 23-11-16

### PR DESCRIPTION
**Pull request by mango-gremlin**
## What was changed/added
Added Prototype Plan Notes for the bridge prototype.

### CODE REVIEW

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+present a prototype primary gameplay loop
+
+### Todo
+- [ ] simplify all mechanics to prototype level
+- [ ] all focus on one mechanic -> bridges
+- [ ] start unterteilen tasks and plan out the weeks with many deadlines so people actually do something
+
+Proposal:
+- 1st week -> code rudimentary drag and drop bridge stuff
+- 2nd week -> debug
```
Comment (ScatteredDrifter) [inline docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md]: and refactor (( to tackle technical debt after we have a functioning prototype ))

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+## Mechanics to be implemented
+
+1. Prototyp 2D Scene mit draggable objects
```
Comment (ScatteredDrifter) [inline docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md]: maybe amend that they need a grid system

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+feature/...
+issue/..
+doc/..
+
+## Weeks until:
+
+### Half Week: 16.-19.11.
+- Godot anschauen, einlesen
+
+##### Bridge Framework
```
Comment (ScatteredDrifter) [inline docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md:36]: (Lernziele)

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- nodes anlegen 
+- als was könnte man draggable Objects/Tiere implementieren
+- wie kann man hitboxen implementieren
+- grid system zum snappen von tieren
+- wie checken, dass puzzle vollendet ist (flooding)
+
+#### Ziele für Sonntag
+- alle programmieren mit, auch game design -> innerhalb dessen Teams für die erstellung dieses Prototyps
+	- für jede Gruppe 1 verantwortlichen, der progress updates an die anderen gibt
+	- man trifft sich dann in diesen Gruppen während der woche, so oft wie nötig
```
Comment (ScatteredDrifter) [inline docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md:48]: "so oft wie nötig", aber mindestens 1x

**Review by ScatteredDrifter — COMMENTED** [review]
passt, was amended mostly already in the commit that follows this

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md (hunk 1 above)
Comment (ScatteredDrifter) [reply docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md]: done in 43a6ef7

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md (hunk 2 above)
Comment (ScatteredDrifter) [reply docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md]: done in 43a6ef7

File: docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md (hunk 4 above)
Comment (ScatteredDrifter) [reply docs/meeting_notes/general_meetings/23-11-16_Prototype-Plan.md]: solved in 303dee5

**Review by marcel-iv — APPROVED** [review]

**Review by ScatteredDrifter — APPROVED** [review]
newest version is fine for me and contains most information

**Pull request merged**